lib/sierra-models/base.js

```diff
@@ -20,110 +21,95 @@ class SierraBase {
 
   fieldTag (field, subfields, opts) {
     const varFields = this._varFieldByFieldTag(field)
-    return this._extractValuesFromVarFields(varFields, subfields, opts)
+    return this._prettifyRawVarFields(varFields, subfields, opts)
```

Comment:
I'm not sure "prettify" is the right term here, but I hear you that "_extractValuesFromVarFields" also doesn't really capture it. "Prettification" tends to mean formatting. What `_prettifyRawVarFields` is doing is transforming raw varfields into VarFieldMatch objects.